test/system/capability.test.js

```diff
@@ -1,16 +1,16 @@
 const test = require('ava')
 const goshawkdb = require('../..')
-const {setupThenTransactionTest, firstRoot} = require('../helpers/utils')
+const {setupThenTransactionTest, testRootName} = require('../helpers/utils')
 
 test("Capabilities are enforced.",
 	setupThenTransactionTest(
 		(t, connection, txn) => {
-			const root = firstRoot(txn)
+			const root = txn.roots[testRootName]
```

Comment:
unfortunately it will mean root is undefined and will error when you try to do something with it.  Probably best way to actually fix this would be to add a txn.root(rootName) function that throws an error if you try to get one that doesn't exist.